launch/kinesis-to-firehose.yml

```diff
@@ -1,10 +1,10 @@
-run:
-  type: docker
-env: []
+env:
+- AWS_ACCESS_KEY_ID
+- AWS_SECRET_ACCESS_KEY
 resources:
-  cpu: 0.1
-  max_mem: 0.1
+  cpu: 1.0
```

Comment:
This is just ballpark numbers, but I'm pretty sure it'll be more than this.

I'll monitor and circle back on this.